libs/threshold-keypair/src/signature.rs

```diff
@@ -134,8 +163,13 @@ pub enum EcdsaSignatureError {
 pub mod tests {
     use super::*;
     use crate::{privatekey::ThresholdPrivateKey, publickey::ThresholdPublicKey};
-    use cggmp21::signing::{DataToSign, Signature};
-    use generic_ec::{coords::AlwaysHasAffineX, curves::Secp256k1, NonZero, Point, SecretScalar};
+    use cggmp21::signing::{DataToSign, Signature as cggmp21Signature};
```

Comment:
For naming conventions reasons:

```suggestion
    use cggmp21::signing::{DataToSign, Signature as Cggmp21Signature};
```